docs/README.md

```diff
@@ -62,6 +62,8 @@ rbenv install 2.6.3
 gem install bundler
 ```
 
+*Note: if you hit a permissions error for the `gem install bundler` follow advice from the [bundler docs](https://bundler.io/doc/troubleshooting.html#permission-denied-when-installing-bundler)*
```

Comment:
This looks good. However, I noticed this link does not talk about the `vendor/cache/` which you had mentioned w/ the first change. Let me know if that was required for you to build the website along with the user install?

Reply:
Turns out that I didn't need to change the vendor cache location.  default seems to be fine.